src/hearings/containers/linked-hearings/linked-hearings-with-case/linked-hearings-with-case.component.ts

```diff
@@ -45,6 +45,8 @@ export class LinkedHearingsWithCaseComponent implements OnInit, OnDestroy {
   public showSpinner: boolean = true;
   public hearingStageOptions: LovRefDataModel[];
 
+  public message: string = 'Hearings may be unavailable due to their status, or if they are already linked to another hearing.\nOnly hearings from cases linked to this case will appear in this list.';
```

Comment:
You've change the strategy for line breaks here. Will this definitely work? Normally a paragraph element would ignore cr/nl characters